src/main/java/com/google/edith/servlets/DataServlet.java

```diff
@@ -10,7 +9,7 @@
 /**
 * Servlet that returns some example content.
 */
-@WebServlet("/api/v1/test-servlet")
+@WebServlet("/")
```

Comment:
Are we sure we want this servlet to link to the home page of the website?

Reply:
for some reason it was giving me a 503 error unless i linked it to home. not sure why, but i never use the servlet so this can be changed to whatever later on

Reply:
can we take this file out of this PR ?
https://git-scm.com/docs/git-checkout#_examples